### PR TITLE
Models

### PR DESCRIPTION
**Pull request by JDziewonski98**
# Summary

added artist and borough tables, and two foreign keys referencing the new tables to mural. wrote some unit tests, would like someone else to expand on them to learn the environment. 

## Test Plan

some unit tests

## Related Issues

implementing the models

### CODE REVIEW

**Review by Tony9984 — APPROVED** [review]
LGTM! just to make sure, we're using the Sequelize ORM due to the Postgre DB right?

File: backend/src/models/tests/artist.test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  expect(artist.id).toEqual(1);
+});
+
+test("get artist", async () => {
+  expect.assertions(3);
+  const artist = await Artist.findByPk(1);
+  expect(artist).not.toEqual(null);
+  expect(artist!.id).toEqual(1);
+  expect(artist!.name).toEqual("testartist");
+});
```
Comment (zhenglinxiao) [inline backend/src/models/tests/artist.test.ts:25]: I thought unit tests were supposed to be written independently from other tests but this one seems to rely on the previous one... Is that okay?

**Review by zhenglinxiao — APPROVED** [review]

File: backend/src/app.ts [hunk 2]
```diff
@@ -2,17 +2,19 @@ import express from "express";
 import bodyParser from "body-parser";
 import "dotenv/config";
 import { Routes } from "./api/routes";
+import { AssociationSetup } from "./models/associationsetup";
```
Comment (tedspare) [inline backend/src/app.ts:5]: Should model names be in camelCase?

File: backend/src/models/mural.model.ts [hunk 3]
```diff
@@ -74,16 +64,14 @@ Mural.init(
   }
 );
 
-Mural.sync({ force: true }).then(() => console.log("Mural table created"));
-
 // this defines what we can see through api calls
 export interface MuralInterface {
   name: string;
-  artist: string;
+  ArtistId: number;
```
Comment (tedspare) [inline backend/src/models/mural.model.ts:70]: camelCase or nah?

**Review by tedspare — APPROVED** [review]
LGTM

File: backend/src/models/tests/artist.test.ts (hunk 1 above)
Comment (JDziewonski98) [reply backend/src/models/tests/artist.test.ts:25]: yes you are right, I was just trying to test as much stuff as possible by the meeting today, but that is best practice, ill change it before merge

**Comment by JDziewonski98** [comment]
Hey @tedspare @yi-fang-yuan @zhenglinxiao @Tony9984 sorry about this PR, I was really hacking it, shouldn't have been changing it so much after opening / had it this big. Also, the prettier code formatting server side is a nightmare lol, gonna make it client side. Anyway, I think another once-over would be good.

File: backend/src/models/tests/artist.test.ts (hunk 1 above)
Comment (JDziewonski98) [reply backend/src/models/tests/artist.test.ts]: changed it to basically clear the database before every test using beforeEach()

**Comment by JDziewonski98** [comment]
> LGTM! just to make sure, we're using the Sequelize ORM due to the Postgre DB right?

Yes, just based on my research sequelize seemed to be the most popular choice for working with Postgres. Was there some other one you recommend? Sequelize seemed to be the most popular for express + postgres, but I've never used it tbh so my progress can be slow...

File: backend/jest.config.js [hunk 4]
```diff
@@ -1,13 +1,11 @@
 module.exports = {
-    "roots": [
```
Comment (JDziewonski98) [inline backend/jest.config.js:2]: just formatted

**Comment by Tony9984** [comment]
> > LGTM! just to make sure, we're using the Sequelize ORM due to the Postgre DB right?
> 
> 
> 
> Yes, just based on my research sequelize seemed to be the most popular choice for working with Postgres. Was there some other one you recommend? Sequelize seemed to be the most popular for express + postgres, but I've never used it tbh so my progress can be slow...

Ohhhh no dude don't worry about it, everything looks super clean when I checked your code haha. Honestly, I forgot we were using Postgre. I was used to using mongoose, but it's a MongoDB ORM. I've never used sequelize as well, but I've looked through it a bit online and everything checks out! It's going to be a good learning experience 😁

**Comment by Tony9984** [comment]
Oh and another thing, just to make things more clean. Try to not make too many commits and also include meaningful messages if ever we want to backtrack (commit that broke the app). From experience, it's usually formatted like so: "TYPE: message". TYPE is usually one of these 3: feat, refactor or fix (might be more). A typical commit message would be for example "refactor: change variable names to camel case". Oh and also make sure to squash and merge once merging! If you guys want, I could make a README on PR/commit syntax, I think it would be useful!

**Pull request merged**